# represent robot state
std_msgs/String state
std_msgs/Int8 robot_id
